.github/workflows/publish.yml

```diff
@@ -21,13 +21,11 @@ jobs:
         uses: "gradio-app/gradio/.github/actions/install-all-deps@main"
         with:
           always_install_pnpm: true
-          node_auth_token: ${{ secrets.NPM_TOKEN }}
-          npm_token: ${{ secrets.NPM_TOKEN }}
           skip_build: 'true'
       - name: Build packages
-        run: | 
+        run: |
           . venv/bin/activate
-          pip install build
+          pip install -U build hatch packaging>=23.2  # packaging>=23.2 is needed to build Lite due to https://github.com/pypa/hatch/issues/1381
```

Comment:
This hatch's issue was solved and the fix has already been released: https://github.com/pypa/hatch/releases/tag/hatchling-v1.23.0
So this `packaging` spec can be removed.

Reply:
hmm, it didn't work well.
https://github.com/gradio-app/gradio/actions/runs/8828922391/job/24238872108

Reply:
Keep installing `packaging` here then.